notes/books/thinking-fast-slow.md

```diff
@@ -0,0 +1,233 @@
+start to engage into tasks that require a lot of system 2 thinking. It is a demanding
+activity for your whole body, that is why we are pretty lazy about it.
+
+## Ego Depletion
+
+When you need to excert discipline and self control, an activity that is system 2,
+you use a lot of energy. It is not as obvious as when you run and feel tired, but
+it is there. Research showed how people after having to excert a lot of self control
+gave up easier in a next challenge than people who were just starting.
+
+This phenomenon is referred as **ego depletion**. When your ego is depleted you will
+be much more prone to make decisions with your System 1, because you are literally
+exausted mentally.
+
+This also applies to when you have to do something that you don't want to do.
+I immediately made a connection between this and meetings. I always found amazing
+how meetings have the ability of completely fuck up my entire day, even when it is
+"just" an hour or two. The fact that I need to force myself to do something
+I don't want to and don't see the point causes burns through the ego, and then
+you have the cost of the interruption itself. In the end it makes perfect sense
+that one or two meetings can destroy a day. It seems exagerated, I always felt
+more sensible than the normal, specially because Im usually the only one complaining.
+But apparently there is some research backing this up and
+[Im not the only one to feel his way](http://paulgraham.com/makersschedule.html).
+
+## Mood and System 1
+
+Some research is mentioned linking the effects of bad mood and System 1.
+When you are in a bad mood your system 1 is impaired. That is why it is hard to
+be creative when you are in a bad mood, have ideas, make associations, etc.
+
+System 2 is much less affected, so you can still perform more analytical activities.
+
+## Biases
+
+Now a list of the biases mentioned throughout the book.
+
+### Consistency and simplicity over accuracy
+
+Our system 1 is predominantly driven by consistency and simplicity in the story
+it tells about ourselves and the world. It doesn't enjoy [cognitive dissonance](https://en.wikipedia.org/wiki/Cognitive_dissonance),
+even though exposing yourself to it constantly is essential to build critical
+thinking.
+
+What does this mean ? It means that when exposed to new information your mind will
+prioritize information that already fits on your current mental model of things,
+because that is comfortable and makes it easy to create a simple and consistent
+narrative about things. If the information is real/accurate is completely irrelevant
+to this process
+
+What does this mean ? It means that when exposed to new information your mind will
+prioritize information that already fits on your current mental model of things,
+because that is comfortable and makes it easy to create a simple and consistent
+narrative about things. If the information is real/accurate is completely irrelevant
+to this process.
+
```

Comment:
Noticed some duplicate text:

```suggestion
What does this mean ? It means that when exposed to new information your mind will
prioritize information that already fits on your current mental model of things,
because that is comfortable and makes it easy to create a simple and consistent
narrative about things. If the information is real/accurate is completely irrelevant
to this process.

```

Reply:
Thanks for the catch ❤️